.eslintrc.json

```diff
@@ -1,6 +1,9 @@
 {
     "extends": "airbnb-base",
     "parser": "@babel/eslint-parser",
+    "parserOptions": {
+        "requireConfigFile": false
```

Comment:
added this because I removed the .babelrc file to favor the configuration through Webpack Encore as from the recipe, see https://github.com/symfony/recipes/blob/main/symfony/webpack-encore-bundle/1.10/webpack.config.js#L54-L57

Reply:
tell me if you prefer it the other way around by keeping the .babelrc file